src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -863,5 +874,111 @@ std::shared_ptr<coreml::MLModelWrapper> drawing_classifier::export_to_coreml(
   return model_wrapper;
 }
 
+void drawing_classifier::import_from_custom_model(variant_map_type model_data,
+                                               size_t version) {
```

Comment:
Let's run `git clang-format` (just on your own changes, please)

Reply:
Oops, my bad. Didn't notice it.